### DISCUSSION TITLE
Al-Furqon TV

### DISCUSSION THREAD

**Discussion by favianrizqulloh** [iptv-org/community#953] [Show and tell]
### Channel ID

AlFurqonTV.id

### Stream URL (optional)

https://stream.alfurqontvgresik.workers.dev/stream.m3u8

### Notes (optional)

- not 24/7
- this channel is part of [ARTVISI](http://streaming.artvisi.or.id/stream/alfurqontv) and [InTech](https://www.lyngsat.com/muxes/Telkom-4_SE-Asia_3773-H.html) satellite network but now no longer broadcasting on satellite since August 1st 2023
- the channel is now live on [YouTube](https://www.youtube.com/channel/UC2FIGD4FVOSWL61jz_yd3HA/live)

### Please confirm the following

- [x] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://h1.intechmedia.net/intech/ch24.m3u8 (shows looped video)
<img width="785" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/f3eeded6-2633-4818-99b4-2da81b6b4f3b">

**Reply by favianrizqulloh** [reply]
Yes it needs user agent spoofing.
Found a way to get around that, same as the other added InTech channels:
`user-agent="Mozilla/5.0 (X11; Linux x86_64)"`

**Comment by favianrizqulloh** [comment]
Hi @freearhey, I've updated the stream link.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
